qcodes/instrument_drivers/Lakeshore/Model_325.py

```diff
@@ -0,0 +1,251 @@
+import numpy as np
+
+from qcodes import VisaInstrument, InstrumentChannel, ChannelList
+from qcodes.utils.validators import Enum, Numbers
+from qcodes.instrument.group_parameter import GroupParameter, Group
+
+
+class Model_325_Channel(InstrumentChannel):
+    """
+    A single sensor channel of a temperature controller
+    """
+
+    sensor_status_codes = {
+        0:  "OK",
+        1:  "invalid reading",
+        16:  "temp underrange",
+        32:  "temp overrange",
+        64:  "sensor units zero",
+        128: "sensor units overrang"
+    }
+
+    def __init__(self, parent, name, channel):
+        super().__init__(parent, name)
+        self._channel = channel
+
+        self.add_parameter(
+            'temperature',
+            get_cmd='KRDG? {}'.format(self._channel),
+            get_parser=float,
+            label='Temerature',
+            unit='K'
+        )
+
+        self.add_parameter(
+            'status',
+            get_cmd='RDGST? {}'.format(self._channel),
+            get_parser=self.decode_sensor_status,
+            label='Sensor_Status'
+        )
+
+        self.add_parameter(
+            "type",
+            val_mapping={
+                "Silicon diode": 0,
+                "GaAlAs diode": 1,
+                "100 Ω platinum/250": 2,
+                "100 Ω platinum/500": 3,
+                "1000 Ω platinum": 4,
```

Comment:
won't "omega" letters be pain in the neck to type in code? I'd substitute it to "Ohm"